base-notebook/test/test_container_options.py

```diff
@@ -71,6 +71,19 @@ def test_sudo(container):
     assert rv == 0
     assert 'uid=0(root)' in c.logs(stdout=True).decode('utf-8')
 
+
+def test_sudo_path(container):
+    """Container should include /opt/conda/bin in the sudo secure_path."""
+    c = container.run(
+        tty=True,
+        user='root',
+        environment=['GRANT_SUDO=yes'],
+        command=['start.sh', 'sudo', 'jupyter', '--version']
```

Comment:
would it be possible to capture the output of ``which jupyter`` and make sure that it starts with ``/opt/conda/bin``? Does that have any advantages?